heroes/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+from rest_framework import routers
+from . import views
```

Comment:
Don't import whole views.
```suggestion
from heroes.views import HeroViewSet
```